data/recipes/gcp_cloud_resource_tree.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "gcp_cloud_resource_tree",
+  "description": "Collects logs from a GCP project and create a resource tree",
```

Comment:
add more description to clarify how this work differently that the offline version, i.e. online mode. because both recipes' descriptions are similar to the cmdline user

Reply:
Done